packages/contracts/hardhat.config.ts

```diff
@@ -11,17 +21,31 @@ export default {
         local: {
             url: 'http://127.0.0.1:8545',
             blockGasLimit: 12000000,
-            accounts: [
-                '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80',
-            ],
+            accounts: [DEFAULT_PRIVATE_KEY],
+        },
+        dev: {
+            url: process.env.ETH_PROVIDER_URL ?? '',
+            accounts: [process.env.PRIVATE_KEY ?? DEFAULT_PRIVATE_KEY],
+        },
+        prod: {
+            url: process.env.ETH_PROVIDER_URL ?? '',
+            accounts: [process.env.PRIVATE_KEY ?? DEFAULT_PRIVATE_KEY],
         },
         arb: {
-            url: 'https://arbitrum.goerli.unirep.io',
-            accounts: [
-                '0x0f70e777f814334daa4456ac32b9a1fdca75ae07f70c2e6cef92679bad06c88b',
-            ],
+            url: process.env.ETH_PROVIDER_URL ?? '',
```

Comment:
should we remove arb config since there's a prod config above?

Reply:
@ryanycw could you check on this, thanks

Reply:
Yes! Agree, I've removed it